doc/source/release-notes.rst

```diff
@@ -20,8 +38,7 @@ Features
 * Make the default timeout configurable per Context and per PV, in addition to
   per a given operation. This makes it possible to adjust all the timeouts in
   one place during debugging.
-* Use a random starting ID for message identifiers as an extra layer of
-  protections against collisions, especially in the context of CI testing where
+* Use a random starting ID for message identifiers as an extra layer of protections against collisions, especially in the context of CI testing where
```

Comment:
What happened to line length limits here?

Reply:
Must have been a typo. Thanks.